lib/livebook/hubs/personal.ex

```diff
@@ -7,6 +7,7 @@ defmodule Livebook.Hubs.Personal do
   alias Livebook.Hubs
 
   @secret_key_size 64
+  @prefix "lb_ph_"
```

Comment:
I would make it more explicit for clarity

```suggestion
  @secret_key_prefix "lb_ph_"
```